transformers/tests/modeling_common_test.py

```diff
@@ -87,12 +87,13 @@ def test_save_load(self):
                 with TemporaryDirectory() as tmpdirname:
                     model.save_pretrained(tmpdirname)
                     model = model_class.from_pretrained(tmpdirname)
+                    model.to(torch_device)
                     with torch.no_grad():
                         after_outputs = model(**inputs_dict)
 
                     # Make sure we don't have nans
-                    out_1 = after_outputs[0].numpy()
-                    out_2 = outputs[0].numpy()
+                    out_1 = after_outputs[0].cpu().numpy()
+                    out_2 = outputs[0].cpu().numpy()
```

Comment:
@aaugustin tensors have to be on cpu to be converted to numpy.

calling .cpu() on a cpu-backed tensor does nothing.